lightning/src/ln/channel.rs

```diff
@@ -1488,6 +1482,12 @@ impl<Signer: Sign> Channel<Signer> {
 		if msg.minimum_depth > config.peer_channel_config_limits.max_minimum_depth {
 			return Err(ChannelError::Close(format!("We consider the minimum depth to be unreasonably large. Expected minimum: ({}). Actual: ({})", config.peer_channel_config_limits.max_minimum_depth, msg.minimum_depth)));
 		}
+		if msg.minimum_depth == 0 {
+			// Note that if this changes we should update the serialization minimum version to
```

Comment:
Following `check_get_funding_locked`, this new check actually prevent the creation of zero-conf chan, which were allowed previous to this PR ? I guess this is intentional waiting for upcoming specification of "turbo-channels" ?

Reply:
Correct, we previously ditn't *handle* the 0-conf option, though. We'd accept a message with a 0 set, but then we'd still wait for a confirmation, so easier to just reject it outright.